content/en/docs/_index.md

```diff
@@ -3,20 +3,136 @@ title: "Documentation"
 linkTitle: "Documentation"
```

Comment:
Note to self - the imported article is a true overview. In the long term we will want this page to serve as a table of contents, so keeping the "Documentation" title is appropriate.